src/sgame/sg_bot_util.cpp

```diff
@@ -1803,6 +1803,8 @@ void BotFireWeaponAI( gentity_t *self )
 	trap_Trace( &trace, muzzle, nullptr, nullptr, targetPos, ENTITYNUM_NONE, MASK_SHOT, 0 );
 	distance = Distance( muzzle, trace.endpos );
 	bool readyFire = self->client->ps.IsWeaponReady();
+	vec3_t dir;
```

Comment:
a position should not be called "dir"

Reply:
Good point, I guess when I did that stuff I didn't had everything clear enough in my mind.